src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,84 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../utils/PersonLink';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slugId } = useParams();
+  const findPersonByName = (name: string) => {
+    return people.find(person => person.name === name);
+  };
```

Comment:
Performance issue: This function is used inside a loop (line 16-25), and inside this function, there is another loop (caused by the `find` method). This leads to quadratic time complexity. It would be better to create a map of people by name before the loop, and use it for constant-time lookups.